sources/packages/forms/src/form-definitions/noticeofassessment.json

```diff
@@ -955,30 +874,31 @@
                       "showWordCount": false,
                       "allowMultipleMasks": false,
                       "addons": [],
-                      "id": "ega6en8"
+                      "id": "ewmwrbl",
+                      "hideLabel": true
                     },
                     {
                       "label": "HTML",
                       "labelWidth": "",
                       "labelMargin": "",
-                      "tag": "span",
-                      "className": "category-header-medium primary-color pt-2",
+                      "tag": "p",
+                      "className": "label-value mt-2",
                       "attrs": [
                         {
                           "attr": "",
                           "value": ""
                         }
                       ],
-                      "content": "${{ data.eligibleAmount }}",
+                      "content": "{{ data.programName }}",
```

Comment:
One and two below seems to be duplicated.

![image](https://github.com/user-attachments/assets/7c09a940-11f0-4a70-a189-e17ed645d69b)